[2m2025-04-19T19:56:12Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]2025/04/19 19:56:12 INFO SSH listening listen_address=[fdaa:15:c794:a7b:148:2ac5:24f4:2]:22
[2m2025-04-19T19:56:13Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]> phonebook-backend@1.0.0 start
[2m2025-04-19T19:56:13Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]> node index.js
[2m2025-04-19T19:56:14Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]Connecting to undefined
[2m2025-04-19T19:56:14Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]Server running on port 3000
[2m2025-04-19T19:56:14Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]error connecting to MongoDB: The `uri` parameter to `openUri()` must be a string, got "undefined". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.
[2m2025-04-19T19:56:15Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]Method: GET
[2m2025-04-19T19:56:15Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]Path:   /installHook.js.map
[2m2025-04-19T19:56:15Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]Body:   undefined
[2m2025-04-19T19:56:15Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]---
[2m2025-04-19T20:00:46Z[0m proxy[784e965b3d5438] [32meze[0m [[34minfo[0m]App part3-phonebook-fly-app-green-sky-2638 has excess capacity, autostopping machine 784e965b3d5438. 1 out of 2 machines left running (region=eze, process group=app)
[2m2025-04-19T20:00:46Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m][32m INFO[0m Sending signal SIGINT to main child process w/ PID 628
[2m2025-04-19T20:00:51Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m][32m INFO[0m Sending signal SIGTERM to main child process w/ PID 628
[2m2025-04-19T20:00:51Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m][32m INFO[0m Main child exited with signal (with signal 'SIGTERM', core dumped? false)
[2m2025-04-19T20:00:51Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m][32m INFO[0m Starting clean up.
[2m2025-04-19T20:00:51Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m][33m WARN[0m could not unmount /rootfs: EINVAL: Invalid argument
[2m2025-04-19T20:00:51Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m][  357.398212] reboot: Restarting system
[2m2025-04-19T20:01:05Z[0m proxy[d891671f419798] [32meze[0m [[34minfo[0m]App part3-phonebook-fly-app-green-sky-2638 has excess capacity, autostopping machine d891671f419798. 0 out of 1 machines left running (region=eze, process group=app)
[2m2025-04-19T20:01:05Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m][32m INFO[0m Sending signal SIGINT to main child process w/ PID 628
[2m2025-04-19T20:01:10Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m][32m INFO[0m Sending signal SIGTERM to main child process w/ PID 628
[2m2025-04-19T20:01:11Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m][32m INFO[0m Main child exited with signal (with signal 'SIGTERM', core dumped? false)
[2m2025-04-19T20:01:11Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m][32m INFO[0m Starting clean up.
[2m2025-04-19T20:01:11Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m][33m WARN[0m could not unmount /rootfs: EINVAL: Invalid argument
[2m2025-04-19T20:01:11Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m][  300.292976] reboot: Restarting system
[2m2025-04-19T20:02:18Z[0m runner[d891671f419798] [32meze[0m [[34minfo[0m]Pulling container image registry.fly.io/part3-phonebook-fly-app-green-sky-2638:deployment-01JS7T96D4J3SAT04Y0HRG51F4
[2m2025-04-19T20:02:18Z[0m runner[784e965b3d5438] [32meze[0m [[34minfo[0m]Pulling container image registry.fly.io/part3-phonebook-fly-app-green-sky-2638:deployment-01JS7T96D4J3SAT04Y0HRG51F4
[2m2025-04-19T20:02:19Z[0m runner[d891671f419798] [32meze[0m [[34minfo[0m]Successfully prepared image registry.fly.io/part3-phonebook-fly-app-green-sky-2638:deployment-01JS7T96D4J3SAT04Y0HRG51F4 (1.323894742s)
[2m2025-04-19T20:02:19Z[0m runner[d891671f419798] [32meze[0m [[34minfo[0m]Configuring firecracker
[2m2025-04-19T20:02:19Z[0m runner[784e965b3d5438] [32meze[0m [[34minfo[0m]Successfully prepared image registry.fly.io/part3-phonebook-fly-app-green-sky-2638:deployment-01JS7T96D4J3SAT04Y0HRG51F4 (1.160479154s)
[2m2025-04-19T20:02:19Z[0m runner[784e965b3d5438] [32meze[0m [[34minfo[0m]Configuring firecracker
[2m2025-04-19T20:02:25Z[0m proxy[784e965b3d5438] [32meze[0m [[34minfo[0m]Starting machine
[2m2025-04-19T20:02:25Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]2025-04-19T20:02:25.775962049 [01JS7TA2B86NFRVBDY45NMNDXC:main] Running Firecracker v1.7.0
[2m2025-04-19T20:02:26Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m][32m INFO[0m Starting init (commit: d15e62a13)...
[2m2025-04-19T20:02:26Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m][32m INFO[0m Preparing to run: `docker-entrypoint.sh npm run start` as root
[2m2025-04-19T20:02:26Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m][32m INFO[0m [fly api proxy] listening at /.fly/api
[2m2025-04-19T20:02:26Z[0m runner[784e965b3d5438] [32meze[0m [[34minfo[0m]Machine started in 1.105s
[2m2025-04-19T20:02:26Z[0m proxy[784e965b3d5438] [32meze[0m [[34minfo[0m]machine started in 1.121973223s
[2m2025-04-19T20:02:27Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]2025/04/19 20:02:27 INFO SSH listening listen_address=[fdaa:15:c794:a7b:146:d4f9:f6cb:2]:22
[2m2025-04-19T20:02:28Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]> phonebook-backend@1.0.0 start
[2m2025-04-19T20:02:28Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]> node index.js
[2m2025-04-19T20:02:29Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]Connecting to undefined
[2m2025-04-19T20:02:29Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]Server running on port 3001
[2m2025-04-19T20:02:29Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]error connecting to MongoDB: The `uri` parameter to `openUri()` must be a string, got "undefined". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.
[2m2025-04-19T20:02:29Z[0m proxy[784e965b3d5438] [32meze[0m [[34minfo[0m]machine became reachable in 3.01884436s
[2m2025-04-19T20:02:31Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]Method: GET
[2m2025-04-19T20:02:31Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]Path:   /api/persons
[2m2025-04-19T20:02:31Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]Body:   undefined
[2m2025-04-19T20:02:31Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]---
[2m2025-04-19T20:02:37Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]Method: GET
[2m2025-04-19T20:02:37Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]Path:   /api/persons
[2m2025-04-19T20:02:37Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]Body:   undefined
[2m2025-04-19T20:02:37Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]---
[2m2025-04-19T20:02:41Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]/app/node_modules/mongoose/lib/drivers/node-mongodb-native/collection.js:187
[2m2025-04-19T20:02:41Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]          const err = new MongooseError(message);
[2m2025-04-19T20:02:41Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]                      ^
[2m2025-04-19T20:02:41Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]MongooseError: Operation `people.find()` buffering timed out after 10000ms
[2m2025-04-19T20:02:41Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]    at Timeout.<anonymous> (/app/node_modules/mongoose/lib/drivers/node-mongodb-native/collection.js:187:23)
[2m2025-04-19T20:02:41Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]    at listOnTimeout (node:internal/timers:594:17)
[2m2025-04-19T20:02:41Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]    at process.processTimers (node:internal/timers:529:7)
[2m2025-04-19T20:02:41Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]Node.js v22.14.0
[2m2025-04-19T20:02:41Z[0m proxy[784e965b3d5438] [32meze[0m [[31merror[0m][PU02] could not complete HTTP request to instance: connection closed before message completed
[2m2025-04-19T20:02:41Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]npm notice
[2m2025-04-19T20:02:41Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]npm notice New major version of npm available! 10.9.2 -> 11.3.0
[2m2025-04-19T20:02:41Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]npm notice Changelog: https://github.com/npm/cli/releases/tag/v11.3.0
[2m2025-04-19T20:02:41Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]npm notice To update run: npm install -g npm@11.3.0
[2m2025-04-19T20:02:41Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]npm notice
[2m2025-04-19T20:02:42Z[0m proxy[d891671f419798] [32meze[0m [[34minfo[0m]Starting machine
[2m2025-04-19T20:02:42Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]2025-04-19T20:02:42.128747266 [01JS7TA26F578XB6RV282NTR9Y:main] Running Firecracker v1.7.0
[2m2025-04-19T20:02:42Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m][32m INFO[0m Starting init (commit: d15e62a13)...
[2m2025-04-19T20:02:42Z[0m proxy[784e965b3d5438] [32meze[0m [[31merror[0m][PC01] instance refused connection. is your app listening on 0.0.0.0:3001? make sure it is not only listening on 127.0.0.1 (hint: look at your startup logs, servers often print the address they are listening on)
[2m2025-04-19T20:02:42Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m][32m INFO[0m Main child exited normally with code: 1
[2m2025-04-19T20:02:42Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m][32m INFO[0m Starting clean up.
[2m2025-04-19T20:02:43Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m][32m INFO[0m Preparing to run: `docker-entrypoint.sh npm run start` as root
[2m2025-04-19T20:02:43Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m][32m INFO[0m [fly api proxy] listening at /.fly/api
[2m2025-04-19T20:02:43Z[0m runner[d891671f419798] [32meze[0m [[34minfo[0m]Machine started in 1.06s
[2m2025-04-19T20:02:43Z[0m proxy[d891671f419798] [32meze[0m [[34minfo[0m]machine started in 1.080151019s
[2m2025-04-19T20:02:43Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m][33m WARN[0m could not unmount /rootfs: EINVAL: Invalid argument
[2m2025-04-19T20:02:43Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m][   17.707171] reboot: Restarting system
[2m2025-04-19T20:02:43Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]2025/04/19 20:02:43 INFO SSH listening listen_address=[fdaa:15:c794:a7b:148:2ac5:24f4:2]:22
[2m2025-04-19T20:02:45Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]> phonebook-backend@1.0.0 start
[2m2025-04-19T20:02:45Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]> node index.js
[2m2025-04-19T20:02:45Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]Connecting to undefined
[2m2025-04-19T20:02:45Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]Server running on port 3001
[2m2025-04-19T20:02:45Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]error connecting to MongoDB: The `uri` parameter to `openUri()` must be a string, got "undefined". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.
[2m2025-04-19T20:02:45Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]2025-04-19T20:02:45.559426686 [01JS7TA2B86NFRVBDY45NMNDXC:main] Running Firecracker v1.7.0
[2m2025-04-19T20:02:46Z[0m proxy[d891671f419798] [32meze[0m [[34minfo[0m]machine became reachable in 3.413685504s
[2m2025-04-19T20:02:46Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]Method: GET
[2m2025-04-19T20:02:46Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]Path:   /installHook.js.map
[2m2025-04-19T20:02:46Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]Body:   undefined
[2m2025-04-19T20:02:46Z[0m app[d891671f419798] [32meze[0m [[34minfo[0m]---
[2m2025-04-19T20:02:46Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m][32m INFO[0m Starting init (commit: d15e62a13)...
[2m2025-04-19T20:02:46Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m][32m INFO[0m Preparing to run: `docker-entrypoint.sh npm run start` as root
[2m2025-04-19T20:02:46Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m][32m INFO[0m [fly api proxy] listening at /.fly/api
[2m2025-04-19T20:02:46Z[0m runner[784e965b3d5438] [32meze[0m [[34minfo[0m]Machine started in 1.113s
[2m2025-04-19T20:02:46Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]2025/04/19 20:02:46 INFO SSH listening listen_address=[fdaa:15:c794:a7b:146:d4f9:f6cb:2]:22
[2m2025-04-19T20:02:48Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]> phonebook-backend@1.0.0 start
[2m2025-04-19T20:02:48Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]> node index.js
[2m2025-04-19T20:02:49Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]Connecting to undefined
[2m2025-04-19T20:02:49Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]Server running on port 3001
[2m2025-04-19T20:02:49Z[0m app[784e965b3d5438] [32meze[0m [[34minfo[0m]error connecting to MongoDB: The `uri` parameter to `openUri()` must be a string, got "undefined". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.
